shorten_service/app.py

```diff
@@ -36,10 +51,15 @@ def lambda_handler(event, context):
            }
 
 
-       # TODO (tarik): Replace this with base62 logic
-       short_id = uuid.uuid4().hex[:7]
-
+       response = short_url_table.update_item(
+           Key={'id': 'counter'},
+           UpdateExpression='ADD url_count :val',
+           ExpressionAttributeValues={':val': Decimal(1)}, # Use Decimal for the counter
+           ReturnValues="UPDATED_NEW"
+       )
+       counter_value = response['Attributes']['url_count']
 
+       short_id = to_base62(counter_value)
  
```

Comment:
This comment is for line 75 where you create the long url. I left a comment about this in the previous PR. The long URL we are sending back suppose to have our domain name not the api gateway name. Think about it this way:

TinyURL probably has a API Gateway with some url like `https://b123abcde4.execute-api.us-west-2.amazonaws.com` but the short urls it returns back are not extension of this url. They are like `tinyurl.com/xyzabc` So you should return your domain name. Since we are not really considering getting a front end domain name you can just make up yours like `tariksurl.com` so you return `tariksurl.com/xyzabc`.